source/slang/slang-ir-byte-address-legalize.cpp

```diff
@@ -74,6 +74,8 @@ struct ByteAddressBufferLegalizationContext
 
     void processGetEquivalentStructuredBuffer(IRInst* inst)
     {
+        m_builder.setInsertBefore(inst);
```

Comment:
The reason for this builder change is otherwise it will point to a destroyed inst?

Reply:
Yes.